src/main/java/uk/gov/companieshouse/api/accounts/interceptor/TransactionInterceptor.java

```diff
@@ -34,6 +35,10 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             String transactionId = pathVariables.get("transactionId");
             ResponseEntity<Transaction> transaction = transactionManager
                     .getTransaction(transactionId, request.getHeader("X-Request-Id"));
+
+            HttpSession session = request.getSession();
+            session.setAttribute("transaction", transaction);
```

Comment:
This PR places the transaction in session scope but https://github.com/companieshouse/company-accounts.api.ch.gov.uk/pull/16 is placing the transaction in the request scope. Can you agree a consistent approach please?

Reply:
Is this still an issue after what Steve will do in his PR?

Reply:
Nope, Steve is going to modify his PR to be consistent with your approach.